rebasehelper/application.py

```diff
@@ -44,6 +44,7 @@
 from rebasehelper.build_log_analyzer import BuildLogAnalyzerMakeError, BuildLogAnalyzerPatchError
 from rebasehelper.versioneer import versioneers_runner
 from rebasehelper import version
+from pkg_resources import parse_version
```

Comment:
This import line should be above local imports (those from `rebasehelper` package). So it should be under `import six` separated by newline.

Reply:
See https://www.python.org/dev/peps/pep-0008/#imports.